content/Proposals/Accepted/202205-mso.md

```diff
@@ -0,0 +1,63 @@
+## Evolution of the Monitoring Stack Operator (MSO)
```

Comment:
> After reading the whole document I would, remove all mentions of MSO and make the document more about the name/scope change.

Not sure if I agree - naming is the less important part.

+1 on motivation summarization, not duplicating content (linking to past enhancement). I would still keep some of the `Why`  section.

Plans for CMO are worth mentioning at least in a shallow way, as it might impact the decision (:

Reply:
I'd also prefer keeping the whys and the extended scope in this document. As you said @JoaoBraveCoding, the original scope and current focus are described in the original proposal for MSO. This is about the possibly adjusted scope of OO in the future.